core/src/main/java/org/apache/calcite/rel/RelDistributions.java

```diff
@@ -88,6 +88,15 @@ private static ImmutableIntList normalizeKeys(Collection<? extends Number> keys)
     return list;
   }
 
+  /** Creates a distribution. */
+  public static RelDistribution distribution(
```

Comment:
Why is this needed?

Reply:
Not necessary,There is already a [PR](https://issues.apache.org/jira/browse/CALCITE-3867) that supports serialization and deserialization.but when I tested the unit test `testExchange`, I got the message below.

java.lang.NullPointerException: no entry for tag collation
java.lang.RuntimeException: java.lang.NullPointerException: no entry for tag collation
	at org.apache.calcite.tools.Frameworks.withPrepare(Frameworks.java:193)
	at org.apache.calcite.tools.Frameworks.withPlanner(Frameworks.java:135)

Reply:
Should I make a new JIRA?